CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1
 
 * Added support for setup and verification of the event handlers through `Setup[Add|Remove]` and `Verify[Add|Remove|All]` (@lepijohnny, #825) 
 
+* Added support for lambda expressions while creating a mock through `new Mock<SomeType>(() => new SomeType("a", "b")` and `repository.Create<SomeType>("a", "b")` (@frblondin, #884)
```

Comment:
* I think it would be great if you mentioned the reason why the new syntax got added, i.e. why it is beneficial when mocking classes without a default parameterless constructor.
* Missing closing parenthesis in `new Mock<SomeType>(() => new SomeType("a", "b")`
* `repository.Create<SomeType>("a", "b")` is a strange example... shouldn't there be a lambda expression somewhere?